dmutils/documents.py

```diff
@@ -116,17 +116,17 @@ def upload_service_documents(uploader, documents_url, service, request_files, se
 
 
 def file_is_not_empty(file_contents):
-    not_empty = len(file_contents.read(1)) > 0
-    file_contents.seek(0)
-    return not_empty
+    return not file_is_empty(file_contents)
 
 
-def file_is_less_than_5mb(file_contents):
-    size_limit = 5400000
-    below_size_limit = len(file_contents.read(size_limit)) < size_limit
+def file_is_empty(file_contents):
+    empty = len(file_contents.read(1)) == 0
     file_contents.seek(0)
+    return empty
 
-    return below_size_limit
+
+def file_is_less_than_5mb(file_contents):
```

Comment:
`FILE_SIZE_LIMIT` could change, so this should probably be called `file_is_not_over_size_limit` or something else more generic like that.  But that would be a breaking change, so let's stick with it for now.